index.html

```diff
@@ -33,30 +31,30 @@ <h2>Седона - небольшой городок в Аризоне,<br>
         <h3>Заинтересовались?</h3>
         <p>Укажите предполагаемые даты поездки, и мы покажем вам лучшие предложения гостиниц в Седоне</p>
         <p>
-          <button type='button'>Поиск гостиницы в Седоне</button>
+          <button type="button">Поиск гостиницы в Седоне</button>
           <!-- Здесь будет форма
           Дата заезда: <time></time><span></span><br>
           Дата выезда: <time></time><span></span><br>
-          Взрослые: <span> </span> Дети: <span></span><br>-->
-          <button type='button'>Найти</button>
+          Взрослые: <span> </span> Дети: <span></span><br>
+          <button type="button">Найти</button>-->
```

Comment:
```suggestion
          <button type="submit">Найти</button>-->
```
Это кнопка относится к форме. Клик по ней отправляет данные формы на сервер.